dist/about.html

```diff
@@ -0,0 +1,190 @@
+                            <ul class="navbar-nav justify-content-end flex-grow-1 pe-3">
+                                <li class="nav-item">
+                                    <a class="nav-link" href="index.html">Home</a>
+                                </li>
+                                <li class="nav-item">
+                                    <a class="nav-link" href="#">Program</a>
+                                </li>
+                                <li class="nav-item">
+                                    <a class="nav-link" href="#">Buy Ticket</a>
+                                </li>
+                                <li class="nav-item">
+                                    <a class="nav-link" href="#">Sponsors</a>
+                                </li>
+                            </ul>
+                        </div>
+                    </div>
+                </div>
+            </nav>
+            <div class="home-div-about">
+                <h4>"Ignite New Ideas"</h4>
+                <h1>StartUp BattleField 2023</h1>
+                <hr class="desktop-hidden">
+                <div class="about-card">
+                    <p>
+                        The original startup conference — stays fresh, relevant and focused on founders, investors
+                        - BattleField delivers the tools, knowledge, and connections to help you make it happen.
+                    </p>
+                </div>
+                <div class="email-about">
+                    <p>Please contact us per Email for any further questions about StartUp BattleField! <span>hackathon2023@startup.org<span></p>
+                </div>
+            </div>
+        </header>
+
+        <section class="logo-about container-fluid">
+            <h3>StartUp BattleField<br>hackathon 2023 logo</h3>
+            <hr>
+            <p>
+                The logo of StartUp BattleField 2023 was decided through the logo competition from <br><span>8. January to 7. January.<span>
+            </p>
+            <img src="/dist/images/startup-battlefield.jpeg" alt="" srcset="">
+        </section>
+
+        <section class="past-about container-fluid">
+            <h3>See the past StartUp Hackathons</h3>
+            <hr>
+            <p>
+                Take a look at the last two StartUp Hackathons which took place in Uganda and in Rwanda.
+            </p>
+            <div class="row g-3">
+                <div class="col col-6 col-md-6 col-sm-12 group1">
+                    <img src="/dist/images/about_2013.png" alt="" srcset="">
+                </div>
+                <div class="col col-6 col-md-6 col-sm-12 group2">
+                    <img class="img2 mb-4" src="/dist/images/about_2011.png" alt="" srcset="">
+                </div>
+            </div>
+        </section>
+
+        <section class="container-fluid sponsors">
+            <h3 class="d-flex justify-content-center">Sponsors</h3>
```

Comment:
- [ ] Kindly use this `#D3D3D3` colour for the `Sponsors` heading to match the provided design. 😊

Reply:
- [ ] @SamTush as previously requested kindly ensure you change the color for sponsors heading from #afafaf to the recommended #D3D3D3.